---
layout: post
title: "Rough Draft #1 - Discussion"
categories: 
tags: 
---


# Writing out the discussion so far

## Outline


* Easy to see different populations, possibly different species
* how each method was useful
   * focus more on how each result connects and shows similar results

* Symbionts are critical parts to the reef ecosystem
* Symbionts and Brevolium are still being classified
* How this fits into the broader tree
* If Eunicea flexuosa has two species of symbiont present in two separate populations, could this mean that there are other corals that have separated populations that also have multiple symbionts?
   * how many coral species vs symbiont species identified
* Brevolium is cited as found in shallow water (Lewis), is the population from deeper water considered deeper than normal? could this be a new finding about brevolium?
* In the Environment column of the mapping file some of each population were labeled from being in the opposite environment, were they mixed?
   * If two corals living next to each other have different species of symbiont what does that signify/lead to?
   * What implications on reef diversity does this have?
   * Would this increase or decrease coral survivability?
       * more diversity = more likely some survive a stressful event
* How are symbionts passed on for Eunicea?
   * parents or environment? (horizontal or vertical)?


The results from this experiment suggest that the two populations of brevolium found at seperate depths within Eunicea flexuosa are distinct species. While occupying the same species of coral. The difference between the two populations is extremely clear with no outliers, and this has led us to consider that these two populations may be two different species. The lineage and diversity of the Brevolium genus is not well described, so the discovery of the separation between two species that occupy the same coral host could lead to some important conclusions.

**SNP counts**
The basis of this analysis was done with population genomics through the use of SNPs. SNPs are single nucleotide variations in a genome of an individual. These SNPs can be very helpful in determining the composition of a population. Individuals within a single population will have many of the same SNPs since they are passed on through generations. Over time as populations become distinct more SNPs will become indicative of each population making the divide between populations clearer. Using programs, populations can be mapped based upon their SNPs. This experiment used the programs, Principal Component Analysis (PCAs), Bayescan, PCAdapt, and Admixture to observe how the populations were segregated. All of these outputted results that were clear and pointed to a very defined difference in population that also gives evidence that these are in fact cryptic species of Brevolium.
In order to successfully analyse SNP data it needs to be filtered first. The Filtering steps that were used took the initial 1415812 SNPs and reduced it count down to 7043 (Table 1(of filtering steps)). The remaining SNPs only accounted for the well defined and clean SNPs. Each filtering step reduced the amount of SNPs without reducing the overall information found within. Once the SNPs were cleaned the programs that analysed could output clear non-messy information.

**PCA**
The Principle Component Analysis (PCA) showed the two populations segregated from each other (Figure 2a) and the Eigenvalues showed that almost all of the difference can be defined by PC1. When a PCA has a very high Eigenvalue for PC1 it means that it can be described by just that single analysis and therefore the difference between populations is clear. The PCAs were run at many different levels of filtering of the original SNPs. All PCAs showed the same population divide. There were two outliers that did not fall into either population. 

**Bayescan and PCAdapt**
Both show the same conclusions as the PCAs. The values

**Distance Tree and NCBI**
Using the SNP data a distance tree was created that clearly shows the genetic difference between the populations. There is a genetic distance of about .3 which has been described in other research as similar to the difference between (Cats and dogs - placeholder example). This makes it clear that these populations should be considered seperate populations. Using NCBI the tree was rooted into the larger brevolium genus and Symbiodinium family.

**Genes from NCBI**
Using the SNP and genetic information from these corals we were able identify which genes are most relevent to the seperation of the shallow and deep population. 

**Environmental implications**
Corals have been well classified and studied since their importance to the ocean's health is well recognized, however the organism that is essential to corals ability to survive has not been given the same attention. Now that we have recognized how diverse the symbionts that provide for corals are we need to grasp a full understanding of how they have evolved and adapted to their environments. Knowledge of how different species of symbionts are able to handle stress events could lead to much more informed conservtion work that takes into consideration what species of symbiont is being used within transplanted coral.


