---
layout: post
title: "Paper Draft #3"
categories: 
tags: 
---

# Rough Draft



# Abstract

Corals rely on a symbiotic dinoflagellate for energy needs. Until recently these symbionts were classified under the genus symbiodinium. However as Genetic and Phylogenetic analysis became more readily used these symbionts are no longer categorized as a single genus but a wide variety of genetically different groups. Brevolium is one of these symbiont groups. The brevolium group can be found in Eunicea flexuosa, a soft octocoral in the Caribbean that has a wider variety of habitable environments, specifically different populations of this coral can be found at varying depths. In this study the Brevolium within Eunicea flexuosa are tested using a genome wide association study to determine how these populations of symbionts differ from each other. Genome wide association tests have not been performed on Symbiodiniaceae species in other studies. The results point to the conclusion that these two populations of symbiont are in fact genetically separate. An FST value of .4, a PCA analysis, and Distance tree all place these two populations as separate species segregated by depth. Using PCAdapt to determine the SNPs under selection allowed us to see that the genes that differ between the two populations are the genes related to reproduction and light harvesting. This showed that the symbionts have different adaptations for varying light levels. Recognizing and identifying specific species of Brevolium that have different sets of adaptations will be an important part of conservation. The genome wide data analysis was extremely clear suggesting this type of analysis has the power to identify cryptic species of closely related Symbiodiniaceae species in future studies.


# Introduction

Corals around the world have established endosymbiotic relationships with dinoflagellates classified in the family Symbiodinium. Corals require the energy Symbiodinium produces through photosynthesis. The symbiotic algae benefits from the relationship by being provided a constant light source and receiving recycled inorganic nutrients. This endosymbiotic pair are reef builders and provide a home for countless marine species. While corals have been well classified, Symbiodinium has not been readily classified. 

Until recently these symbionts were classified under a single genus called Symbiodinium. However, as we further study this symbiont a diverse group of organisms appear. As a result, the genus Symbiodinium has been split into many genera (Cladocopium, Brevolium, Fugacium, Durusdinium, Greakladium, Effrenium, Symbiodinium) (LaJeunesse 2018) all now under the family Symbiodinium. The species within these genera are not yet fully characterized. Recent studies have suggested that the diversity of the Symbiodinium family is far greater than currently estimated.

The soft coral Eunicea flexuosa is commonly associated with the Brevolium family of symbionts. E. flexuosa is an octocoral from the family Plexauridae. It is a broadcast spawner and acquires its symbiont horizontally from the environment. Horizontal acquisition allows for the coral to have different symbionts between generations.This horizontal acquisition means that while the corals may spread beyond its native home, the symbiont will only spread through proximity. Symbionts therefore have become more adapted to the environment they originate from, adapting to the depth, light levels, and pressure of its environment. Only recently have we recognized the wide variety of morphological, physiological, genetic and physical characteristics within Symbiodinium.

Brevolium has been shown to be a diverse genus with species spread across the Caribbean (Lewis 2018). These putative species still need further testing to identify not only unique species but also the unique traits between species. Understanding the specific environmental conditions symbiont species are able to survive in will be a crucial component of future conservation of coral reefs. Most studies have used morphological or molecular markers to classify species within Symbiodinium. While these methods are useful, we propose that a genome wide analysis of SNPs is an option that will identify the separation of species within Symbiodinium as well as the traits that set apart these species better than previous methods.  

In this paper, we use a population genetics approach to determine the genetic difference between populations of Brevolium. Using Genome wide analysis of SNPs we determine the distance between populations genetically. Using Genome wide SNP analyses is becoming increasingly important in speciation studies (examples). Prior studies have observed a diversity of Symbiodiniaceae greater than currently cataloged, however this will be one of the first to use genome wide association. 

Using the Single Nucleotide Polymorphisms (SNPs) we are also able to study which genes might be responsible for the difference in populations of Brevolium. Programs like Bayescan, PCAdapt, and FstHet can be used to find SNPs under selection. Gene Ontology assists in determining what traits may be the source of selection between the populations. 

Understanding the differences between Brevolium species, and symbiont species in general is becoming increasingly important as conservation efforts try to find possible solutions for coral degradation as ocean temperatures rise. Knowing the symbionts that are best adapted for certain temperatures or depths can help select the best candidates for reseeding of reefs with corals.


# Methods


Original sampling occurred in the Bahamas, Panama, Puerto Rico, and Curaçao  (Prada 2013). Eunicea flexuosa samples were taken at a shallow and a deep depth at each site. The depths of the coral from each site were recorded and used to separate them into two populations. The Brevolium within the coral was genetically sequenced.
The genetic information was transitioned from BAM files to a vcf file using SNP calling. The SNP count following alignment and initial analysis was ~ 1766850. The data file containing the SNPs was filtered using 4 vcftools functions (source). First any SNPs containing indels were removed. The SNPs were then filtered to only contain biallelic, *excluding the heterozygous and homozygous SNPs*. A minimum stack depth of 10 was applied. Anything with over 50% missing info or a minor allele frequency of .01 was removed. Finally the linkage disequilibrium was removed by thinning so that no two sites were within 1000 bp of each other. Principal component analysis (PCA) was run on the filtered vcf file. Using vcfR (reference) the vcf file was combined with a mapping file continuing population information and turned into a genlight object. This genlight object was used to create a distance tree, PCA plot, Eigenvalues plot, DAPC, and compoplot.
A Hardy weinberg filter was applied using vcftools to further filter the data based on their populations.
Using Admixture 8 files with K values from 1 to 8 were generated and the file with the least cross-validation error was selected. Using this file and the mapping file barplots were created for each of the K values.
A Weir and Cockerham FST calculation was performed using vcftools.
Bayescan, PCAdapt, and FstHet analyses were performed in order to determine which SNPs were under selection and which were neutral SNPs. Using Gene Ontology the functions for each SNP under selection was determined.
Finally a phylogenetic tree was created that rooted the two species we have examined into the larger Brevlium tree. 



# Results

## Figures

### Distance tree
![DistnceTree]({{ site.baseurl }}/images/DistanceTree_noNA.pdf "Distance Tree")

**Figure 1**. Phylogenetic analysis of two populations of Brevolium From the Carribean. The populations are physically separated by a depth kline. Genetically they are separated by a distance of approximately .3 or 30% genetic variance.



### PCA

|**(A)**|**(B)**|
|![PCA]({{ site.baseurl }}/images/10x_PCA_noNA.pdf "PCA") | ![Eigenvalues]({{ site.baseurl }}/images/PCA_Eigenvalues_noNA.pdf "Eigenvalues") |

**Figure 2**. **(A)**Principal Component Analysis using PCA1 and PCA2. The two populations are segregated by PCA1. **(B)** The Eigenvalues showing the amount of variance described by each PCA. PCA1 describes most of the variance

### K3 Admixture

![K3]({{ site.baseurl }}/images/Admixture_K3_plot.pdf "Admixture K3 plot")

**Figure 3** An Admixture plot showing the K = 3 values. Assuming that there are three populations.


**FST value was 0.40116**

* NCBI tree rooting our population into larger tree
* Value Tables?
   * Filtering steps
   * Bayescan/PCAdapt/FstHet
	* table of GO functions for SNPs under selection


# Discussion

The results from this experiment suggest that the two populations of brevolium found at seperate depths within Eunicea flexuosa are likely distinct species. While occupying the same species of coral. The lineage and diversity of the Brevolium genus is not well described and the discovery of a separation in species will help to get a clearer understanding of the genus. This study uses a promising analytical method that has not been considered in previous studies. 

**SNP counts**
The basis of this analysis was done with genome wide population genomics through the use of SNPs. SNPs are single nucleotide variations in a genome of an individual. SNPs can be very helpful in determining the composition of a population. Individuals within a single population will have many of the same SNPs since they are passed on through generations. Over time as populations become distinct more SNPs will become indicative of each population making the divide between populations clearer. Using programs, populations can be mapped based upon their SNPs. This experiment used the programs, Principal Component Analysis (PCAs), Bayescan, PCAdapt, FstHet, and Admixture to observe how the populations were segregated. All of these outputted results pointed to a very defined difference in population that could suggest that these populations are in fact cryptic species of Brevolium.
In order to successfully analyse SNP data it needs to be filtered first. The Filtering steps that were used took the initial 1415812 SNPs and reduced it count down to 7043 (Table 1(of filtering steps)). The remaining SNPs only accounted for the well defined and clean SNPs. Each filtering step reduced the amount of SNPs without reducing the overall information found within. 

**PCA**
The Principle Component Analysis (PCA) showed the two populations segregated from each other (Figure 2a) and the Eigenvalues showed that almost all of the difference can be defined by PC1. When a PCA has a very high Eigenvalue for PC1 it means that it can be described by just that single analysis. The PCAs were repeated on files that had less filtering. All PCAs showed the same population divide. There were two outliers that did not fall into either population.

**Admixture**
Admixture is a program that will create a compopolot that separates SNPs into different numbers of populations. This program outputted 8 possibilities ranging from one population to eight populations. each was given a value that indicates the likelihood that this model was accurate. The lowest, and therefore most accurate, model was the model indicating two populations. Figure 3 shows The model indicating three populations. The green bars in the center are signifying the two outliers seen before that do not fall into either population. The deep and shallow populations' bars are almost entirely solid colors indicating they have almost no crossover, and that it is very high certainty that they are separate.

**Bayescan, FstHet, and PCAdapt**
The programs Bayescan, FstHet and PCAdapt use the Fst values to find SNPs that are under selection. The SNPs that are under selection have a selective pressure applied to them and should be associated with genes that make the Brevolium species better adapted to either the shallow or deep population. 

Bayescan uses a program that has a straightforward strict filter. Bayescan did not find SNPs under selection. PCAdapt has broader parameters and discovered 61 SNPs under selection out of the 7877 SNPs. The final program run was FstHet. FstHet discovered 6 SNPs under selection. Three of the SNPs discovered matched with SNPs from the PCAdapt analysis. 

Using Gene Ontology the functions of the contigs associated with each SNP that was found to be under selection was determined. The three genes that were selected by both PCAdat and FstHet were comp9952_c0_seq1 and two SNPs within comp9986_c0_seq1. The comp9952_c0_seq1 gene is associated with ‘Proteasome subunit beta type-1’. This subunit is a component of a complex that removes damaged proteins from cells that may impair cellular function. For Brevolium, this could be associated with the removal of proteins damaged by ultraviolet light. Comp9986_c0_seq1 is associated with Patched domain-containing protein 3 which has been associated with sperm production. Genes of particular interest that were selected by only PCAdapt were GO:E0WED0 and GO:E0WEC8 which both code for a light harvesting protein. GO: Q40297 which codes for a chlorophyll binding protein. GO:P26182 codes for actin and this has been determined to be an alternative molecular marker that can be used to identify Symbiodinium species (X. Pochon 2012). 
*look for genes related to protection and repairment*



**Distance Tree and NCBI**
Using the SNP data a distance tree was created that clearly shows the genetic difference between the populations. There is a genetic distance of about .3 which has been described in other research as similar to the difference between (Cats and dogs - placeholder example). This makes it clear that these populations should be considered separate populations. Using NCBI the tree was rooted into the larger brevolium genus and Symbiodinium family.
The genetic difference between these two populations of symbiont is far greater than the genetic difference between the two populations of E. flexuosa they are associated with.

**FST**
The FST value of 0.40116 is further supports that these two populations are distinct species. In other studies a FST value ot .3 has been used to describe the difference between bird species (Lewis?).

**Environmental implications**
Corals have been well classified and studied due to their importance to the ocean’s ecosystem, however the symbionts within coral that are essential to corals ability to survive have not been given the same attention. Now that we have recognized how diverse the symbionts are that provide for corals, we need to grasp a full understanding of how they have evolved and adapted to their environments. Knowledge of how different species of symbionts are able to handle stress events could lead to much more informed conservation work that takes into consideration what species of symbiont is being used within transplanted coral. We have shown that genome wide SNP analyses can provide definitive results on both speciation between Symbionts and also determine the traits that have been selected for between populations.


# Conclusion

The two populations of Brevolium found within Eunicea flexuosa at different depths have been identified as likely candidates to be cryptic species based upon the population genomics used in this experiment. Further the genes that separate these populations have been identified to be associated with reproduction and light reception. The evidence that there are two species of symbiont living in close proximity to each other within a single species of coral suggests that there could be a much higher magnitude of symbiont species that recently suggestested. This Study has also shown the effectiveness of genome wide analysis, a method previously untested.
