---
layout: post
title: "Rough Draft #1 - Discussion"
categories: 
tags: 
---


# Writing out the discussion so far

## outline

* Easy to see different populations, possibly different species
* how each method was useful or unuseful
    * focs more on how each result connects and shows similar results 

* Symbionts are critical parts to the reef ecosystem 
* Symbionts and Brevolium are still being classified
* How this fits into the broader tree
* If Eunicea flexousa has two species of symbiont present in two seperate populations, could this mean that there are other corals that have sepereated populations that also have multiple symbionts? 
    * how many coral species vs symbiont species identified 
* Brevolium is cited as found in shallow water (Lewis), is the population from deeper water considered deeper than normal? could this be a new finding about brevolium?
* In the ENvironment column of the mapping file some of each population were labeled from being in the opposite environmet, were they mixed? 
    * If two corals living next to each other have different species of symbiont what does that signify/lead to?
    * what implications on reef diversity does this have?
    * Would this increase or decrease coral survivability?
        * more diversity = more likly some survive a stressful event
* how are symbionts passed on for Eunicea? 
    * parents or environment? (horizontal or verticle)?


The results from this experiment suggest that the two populations of brevolium found at seperate depths within Eunicea flexuosa are distinct species. While occupying the same species of coral. The difference between the two populations is extremely clear with no outliers, and this has led us to consider that these two populations may be two different species. The lineage and diversity of the Brevolium genus is not well described, so the discovery of the seperation between two species that occupy the same coral host could lead to some important conclusions.

**SNP counts**
The basis of this analysis was done with population genomics through the use of SNPs. SNPs are single nucleotide variations in a genome of an individule. These SNPs can be very helpful in determining the composition of a population. Individules within a single population will have many of the same SNPs since they are passed on through generations. Over time as populations become distinct more SNPs will become indicitive of each population making the divide between populations clearer. Using programs, populations can be mapped based upon their SNPs. This experiment used the programs, Principle Component Analysis (PCAs), Bayescan, PCAdapt, and Admixture to observe how the populations were segregated. All of these outputted results that were clear and pointed to a very difined difference in population that also gives evidence that these are infact cryptic species of Brevolium.
In order to succsesfully analyse SNP data it needs to be filtered first. The Filtering steps that were used took the initial 1415812 SNPs and reduced it count down to 7043 (Table 1(of filtering steps)). The remaining SNPs only accounted for the well defined and clean SNPs. Each filtering step reduced the ammount of SNPs without reducing the overall information found within. Once the SNPs were cleaned the programs that analysed could output clear non-messy information.

**PCA**
The Principle Component Analysis (PCA) 