---
layout: post
title: "Summary"
categories: 
tags: 
---


# What I have done over the past two weeks

After opservations on the past graphs and SNP counts I refiltered the vcf I was working with in order to keep more SNPS.

|**Filter**|**Code**|**Number of  SNPs**|
|Depth Filter| -min-meanDP 10 |kept 347031 out of a possible 1415812 Sites|
|Missing info | –max-missing 0.5 –minQ 30 | 48145 out of a possible 347031 Sites |
|Minor Allele frequency | –maf 0.01 | 30795 out of a possible 48145 Sites |
|Thinning | –thin 1000 | kept 7877 out of a possible 30795 Sites |
|Hardey Weinberg| perl filter_hwe_by_pop.pl | Kept 7043 of a possible 7877 loci |

The two samples that were labled as NA were also removed

## I created graphs using this newly filtered data set

### Distance Tree
![DistnceTree]({{ site.baseurl }}/images/DistanceTree_noNA.pdf "Distance Tree")

### PCA
![PCA]({{ site.baseurl }}/images/10x_PCA_noNA.pdf "PCA")

### Compoplot
![compoplot]({{ site.baseurl }}/images/Compoplot.pdf "compoplot")



# Bayescan

We then looked for SNPs under selection using Bayescan.

Bayesan did not identify any outliers when run. 


![Bayescanlog10]({{ site.baseurl }}/images/Bayescan_pop_log10.pdf "Bayescan")

|Alpha|Fst1|LogL|
|![Bayescanalpha]({{ site.baseurl }}/images/Bayescan_pop_alpha.pdf "Bayescan pop alpha") |![Bayescanalpha]({{ site.baseurl }}/images/Bayescan_pop_Fst1.pdf "Bayescan pop Fst1") |![BayescanLogL]({{ site.baseurl }}/images/Bayescan_pop_LogL.pdf "Bayescan pop LogL")|



# PCAdapt

After running the PCAdapt the file brevolium_snps_biallelic_10x_mac3_miss50_maf01_thin1000_noNA.recode.vcf was shown to have **61 outliers** out of the total 7877 SNPs

| Neutral SNPs | SNPs under selection |
|![PCAdapt]({{ site.baseurl }}/images/pcadapt_screenplot_outliers.pdf "pcadapt") | ![PCAdapt]({{ site.baseurl }}/images/pcadapt_screenplot_under.pdf "pcadapt") |

|![PCAdapt]({{ site.baseurl }}/images/pcadapt_scores-1,2_outliers.pdf "pcadapt") | ![PCAdapt]({{ site.baseurl }}/images/pcadapt_scores-1,2_under.pdf "pcadapt") |

|![PCAdapt]({{ site.baseurl }}/images/pcadapt_scores-3,4_outliers.pdf "pcadapt") | ![PCAdapt]({{ site.baseurl }}/images/pcadapt_scores-3,4_under.pdf "pcadapt") |

|![PCAdapt]({{ site.baseurl }}/images/pcadapt_Manhattan_outliers.pdf "pcadapt") | ![PCAdapt]({{ site.baseurl }}/images/pcadapt_manhattan_under.pdf "pcadapt") |

|![PCAdapt]({{ site.baseurl }}/images/pcadapt_qqplot_outliers.pdf "pcadapt") |![PCAdapt]({{ site.baseurl }}/images/pcadapt_qqplot_under.pdf "pcadapt") |

|![PCAdapt]({{ site.baseurl }}/images/pcadapt_p-values_outliers.pdf "pcadapt") |![PCAdapt]({{ site.baseurl }}/images/pcadapt_p-values_under.pdf "pcadapt")|

|![PCAdapt]({{ site.baseurl }}/images/pcadapt_stats_outliers.pdf "pcadapt") | ![PCAdapt]({{ site.baseurl }}/images/pcadapt_stats_under.pdf "pcadapt") |



# What I will do next

I will use **Admixture** to show the amount of crossover between the populations 

Use **NCBI** to identify the genes that distinguish the populations from each other

Continue to write practice drafts of a paper.
Write a **Materials and methods** section, an **abstract**, and **figure discriptions** for key graphs.


## Questions